.github/workflows/test-application.yaml

```diff
@@ -61,7 +61,59 @@ jobs:
             - name: Run php-cs-fixer
               run: ./${{ steps.php-cs-fixer-path.outputs.path }} fix --dry-run --diff
 
-    test:
+    js-css:
+        name: Test JavaScript application
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure Node
+              uses: actions/setup-node@v2-beta
+              with:
+                  node-version: '12'
+
+            - name: Get npm cache directory
+              id: npm-cache-dir
+              run: echo "::set-output name=dir::$(npm config get cache)"
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.npm-cache-dir.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/package-lock.json') }}
+
+            - name: Assert error when using yarn
+              run: tests/js/check-yarn-warning.sh
+
+            - name: Install dependencies
+              run: npm install --ignore-optional
+
+            - name: Lint JavaScript and CSS code
+              run: |
+                  npm run lint:js
+                  npm run lint:scss
+                  npm run flow -- check
+                  npm run depcruise
+
+            - name: Test JavaScript code
+              run: npm test -- --maxWorkers=4
+
+            - name: Test Styleguidist build
+              run: npm run styleguide:build
```

Comment:
Is this really necessary since there is an additional workflow building the styleguide for `jsdocs.sulu.io`?

Reply:
i think so - the `jsdocs.sulu.io` action is not executed for pull request, right? 🙂

Reply:
ah yes, you're right 👍